tests/components/notify/test_command_line.py

```diff
@@ -40,6 +41,7 @@ def test_bad_config(self):
 
     def test_command_line_output(self):
         """Test the command line output."""
+        ha.SERVICE_CALL_LIMIT = 1
```

Comment:
Patching the object did not work when it was there before, because that puts in a callable object, while SERVICE_CALL_LIMIT is supposed to be a number. Code trying to do a comparison between a Mock callable object and an integer was throwing an error, I'm not sure why it would've worked in the past.

It doesn't mean that it waits till the end of the value, it means it waits no more than that value. It's a time-out on the longest it will wait. It still completes quite quickly, depending on the system and what Python decides to do with context switching, but it took more than the 0.01 seconds it was set to (which I set in an earlier test that carried through here, I will fix that next).